src/App.tsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react';
-import './App.scss';
-import { MoviesList } from './components/MoviesList';
+import React, { useState } from 'react';
 import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList';
+import './App.scss';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer
+    .filter(movie => movie.title.toLowerCase()
+      .includes(query.toLowerCase()) || movie.description.toLowerCase()
+      .includes(query.toLowerCase()));
```

Comment:
move `query.toLowerCase()` to separate variable instead of recalculate it twice on every iteration
then use this variable here